weldx/asdf/tags/weldx/core/transformations/rotation.py

```diff
@@ -100,11 +100,7 @@ def from_tree(cls, tree, ctx):
             return WXRotation.from_rotvec(tree["rotvec"])
         elif "angles" in tree:
             if "degree" in str(tree["angles"].units):
-                angles = tree["angles"].to("degree").magnitude
-                degrees = True
+                angles = tree["angles"].to("degree")
             else:
```

Comment:
actually the other branch should be explicit, right?

Reply:
not sure what you mean here, can you just post the code snippet?